.github/workflows/rust.yml

```diff
@@ -77,6 +77,19 @@ jobs:
           nix_path: nixpkgs=channel:nixos-22.05
       - uses: cachix/cachix-action@v10
         with:
-          name: mini
-          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+          name: fedimint
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
       - run: nix-build
+  nix-flake:
+    name: Nix-flake Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: cachix/install-nix-action@v16
+        with:
+          nix_path: nixpkgs=channel:nixos-22.05
+      - uses: cachix/cachix-action@v10
+        with:
+          name: fedimint
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+      - run: nix build
```

Comment:
Are `nix-build` and `nix build` actually doing different things?

Reply:
`nix-build` is for building default.nix and `nix build` is for building flake.nix